src/sqs/index.ts

```diff
@@ -258,23 +311,67 @@ export class SqsQueue<MessagePayload> {
 			executionId: uuidv4(),
 			queue: this.queueName})
 
-		const context: Context<MessagePayload> = {message, payload, log }
+		const maybeReceivedCount: MessageAttributeValue | undefined = message.MessageAttributes ? message.MessageAttributes["ApproximateReceiveCount"] : undefined;
+
+		const receiveCount = maybeReceivedCount ? parseInt(maybeReceivedCount?.StringValue || "1") : 1
 
-		log.info("Sqs message received");
+		const context: Context<MessagePayload> = {message, payload, log, receiveCount: receiveCount}
+
+		log.info(`Sqs message received. Receive count: ${receiveCount}`);
 
 		try {
 			const messageProcessingStartTime = new Date().getTime();
-			await this.messageHandler.handle(context)
+
+			// Change message visibility timeout to the max processing time
+			// plus some room for error handling in case of timeout
+			await this.changeVisabilityTimeout(message, this.timeoutSec + 2, log);
```

Comment:
`+ 2` should be a constant.  Why 2 seconds?

Reply:
There is a comment above explaining it. I will add a constant